internal/handlers/pg/msg_hostinfo.go

```diff
@@ -63,7 +62,7 @@ func (h *Handler) MsgHostInfo(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg
 				"cpuArch", runtime.GOARCH,
 			)),
 			"os", must.NotFail(types.NewDocument(
-				"type", strings.Title(runtime.GOOS), //nolint:staticcheck // good enough for GOOS
```

Comment:
fixed.

Reply:
Please revert those changes

Reply:
done

Reply:
This PR replaces
```
strings.Title(runtime.GOOS), //nolint:staticcheck // good enough for GOOS
```
with
```
strings.Title(runtime.GOOS),
```
and now CI fails. If you revert that change, it will pass.

Reply:
added